alchemy/src/cloudflare/nextjs/plugin.ts

```diff
@@ -0,0 +1,7 @@
+import { initOpenNextCloudflareForDev } from "@opennextjs/cloudflare";
+import type { GetPlatformProxyOptions } from "wrangler";
+import { getPlatformProxyOptions } from "../cloudflare-env-proxy.ts";
+
+export function initAlchemyNextjs(options: GetPlatformProxyOptions = {}) {
```

Comment:
Open to feedback on this name here. Would be really nice if it was just a plugin you could throw inside your `next.config.ts` but nope.

Note: `initOpenNextCloudflareForDev` is async but doesn't need to be awaited, per their docs. We can mention that in a comment.